hibernate-core/src/main/java/org/hibernate/metamodel/mapping/internal/ToOneAttributeMapping.java

```diff
@@ -140,7 +141,7 @@
 	private final NotFoundAction notFoundAction;
 	private final boolean unwrapProxy;
 	private final boolean isOptional;
-	private final EntityMappingType entityMappingType;
+	private final EntityPersister entityMappingType;
 
```

Comment:
This could have been left as EntityMappingType (which EntityPersister extends btw).  I'd rather we start relying on the new contracts when possible.  The persister is also available when needed via `EntityMappingType#getEntityPersister`

Reply:
Reverted back to `EntityMappingType`!